src/wp-includes/block-template-utils.php

```diff
@@ -317,6 +338,17 @@ function _get_block_templates_files( $template_type ) {
 				// Subtract ending '.html'.
 				-5
 			);
+
+			// Skip this item if its slug doesn't match any of the slugs to include.
+			if ( ! empty( $slugs_to_include ) && ( false === array_search( $template_slug, $slugs_to_include, true ) ) ) {
```

Comment:
```suggestion
			if ( ! empty( $slugs_to_include ) && ! isset( $slugs_to_include[ $template_slug ] ) ) {
```

Instead of `array_search` use `isset` to check for the presence.

Reply:
_Edit: sorry, accidentally merged this one._

So, `$slugs_to_include` is going to be an array such as `array( 'some_slug', 'some_other_slug' )`. Using `isset` in this case will always return `false`, whether or not the key exists. Unless am I missing something?

Reply:
```suggestion
			if ( ! empty( $slugs_to_include ) && ! in_array( $template_slug, $slugs_to_include, true ) ) {
```

Good catch.  In that case, `in_array` is more performant than `array_search`, so use it.

Reply:
Happy to use `in_array` if that's preferred, though I doubt it has any performant impact in this scenario.